stdlib/src/sys/intrinsics.mojo

```diff
@@ -1504,6 +1504,34 @@ struct _RegisterPackType[*a: AnyTrivialRegType]:
         return __mlir_op.`kgen.pack.extract`[index = i.value](self.storage)
 
 
+# ===----------------------------------------------------------------------=== #
+# expect
+# ===----------------------------------------------------------------------=== #
+
+
+@always_inline("nodebug")
+fn expect[T: AnyTrivialRegType, //, expected_val: T](val: T) -> T:
```

Comment:
Deleted a previous comment about this, I thought the positional params issue was not resolved and it seems like it is if it's passing the tests :+1: solid addition